src/library/Model/ServiceApiKey.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Copyright 2022-2023 FOSSBilling
+ * Copyright 2011-2021 BoxBilling, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * @copyright FOSSBilling (https://www.fossbilling.org)
+ * @license http://www.apache.org/licenses/LICENSE-2.0 Apache-2.0
+ */
+
+class Model_ServiceApiKey extends \RedBeanPHP\SimpleModel
```

Comment:
Something to note:
Because this module doesn't use the deprecated logic that the existing "service" module's do, it doesn't actually seem like this model is used at all. If/when we migrate to a different ORM, this will still be useful, but even if we don't it also means that anyone can go and look at what the properties are and their types without needing to look at the SQL structure for it.